jboss/container/maven/36/36scl/module.yaml

```diff
@@ -0,0 +1,22 @@
+schema_version: 1
+name: jboss.container.maven.36.bash
+version: '3.6scl'
+description: Provides Maven v3.6 capabilities to an image via SCL
+
+labels:
+- name: io.fabric8.s2i.version.maven
+  value: "3.6"
+
+envs:
+- name: JBOSS_CONTAINER_MAVEN_36_MODULE
+  value: /opt/jboss/container/maven/36/
+- name: MAVEN_VERSION
+  value: "3.6"
+
+execute:
+- script: configure.sh
+
+packages:
+  install:
+  - rh-maven36
```

Comment:
@jmtd Will rh-maven36 ever even exist? I suspect we don't / won't need 36scl. I'm not 100% sure about this though.

Reply:
It is currently being QA'd

Reply:
The combination of RHEL7, JDK8 and Maven 3.6 is imporant for Quarkus, alas.